technical-reference/reference-technical-articles/automatic-income-allocation.md

```diff
@@ -0,0 +1,87 @@
+# Automatic income allocation
+
+The purpose of this document is to define the technical solution for distributing the incomes generated by the Venus protocol. The distribution takes into account the incomes generated by the spread (borrow minus supply rates) and liquidations (part of the liquidation incentives). The document covers the distribution process for both Core pools and Isolated pools, and explains how the incomes are allocated to different destinations based on specific rules and percentages. Additionally, it outlines the approach for near-streaming income distribution.
+
+## Income Distribution Schema
+We want to distribute the protocol incomes this way *[Tokenomics](https://snapshot.org/#/venus-xvs.eth/proposal/0xc9d270ccecb7b91c75b95b8d9af24fc7c20cd38c0c0c44888ed4e7724f4e7ce9)*
+
+The Venus Protocol generates income from two main sources:
+
+- (A) Spread between the borrowing rate and the supply rate
+- (B) Part of the liquidation incentives
+
+The distribution of protocol incomes is as follows:
+
+**Schema 1:**
+
+- When: Income generated by (A) from markets supported by the Prime program.
+- Distribution:
+    - 40%: Risk fund
+    - 40%: Treasury
+    - 10%: XVS Vault rewards
+    - 10%: Venus Prime
+
+**Schema 2:**
+
+- When: Conditions for Schema 1 are not satisfied.
+- Distribution:
+    - 50%: Risk fund
+    - 40%: Treasury
+    - 10%: XVS Vault rewards
+
+The initial distribution excludes VAI funds, which are kept in the VTreasury.
+
+## Streaming Income Distribution
+
+<figure><img src="../.gitbook/assets/autoamtic-income-allocation.png" alt=""></figure>
+
+The Venus protocol aims to distribute accumulated reserves in the markets automatically and in near-real-time, leveraging transactions executed by users. This approach eliminates the need for external tools to manage the distribution process. . The `ProtocolShareReserve` contract serves as the designated destination for liquidation incomes, allowing for tracking and distribution within the Venus Protocol.
+
+### Core Pool - Spread Income
+
+In Core pools, accumulated reserves in the markets will be distributed after reaching a specified threshold. The threshold is based on the number of blocks since the last transfer of reserves. This strategy helps socialize part of the cost associated with distribution.
```

Comment:
```suggestion
In the Core Pool, accumulated reserves in the markets will be distributed after reaching a specified threshold. The threshold is based on the number of blocks since the last transfer of reserves. This strategy helps socialize part of the cost associated with distribution.
```